lib/compat/wordpress-6.0/class-gutenberg-rest-global-styles-controller.php

```diff
@@ -81,7 +81,8 @@ public function get_theme_items( $request ) {
 		$base_directory = get_stylesheet_directory() . '/styles';
 		if ( is_dir( $base_directory ) ) {
 			$nested_files      = new RecursiveIteratorIterator( new RecursiveDirectoryIterator( $base_directory ) );
-			$nested_html_files = new RegexIterator( $nested_files, '/^.+\.json$/i', RecursiveRegexIterator::GET_MATCH );
+			$nested_html_files = iterator_to_array( new RegexIterator( $nested_files, '/^.+\.json$/i', RecursiveRegexIterator::GET_MATCH ) );
+			ksort( $nested_html_files );
```

Comment:
What was this change about?

Reply:
Hi @youknowriad, right now as variations don't have names we rely on the order the variations appear on the UI as a way of selecting a variation. We were not sorting the variations so the order was different from time to time and tests failed sometimes. This change makes sure the variations are alphabetically sorted by the file name on the REST response and as a consequence on the UI.